### ISSUE TITLE
NIP-95 v2 feedback

### ISSUE THREAD

**Issue by arthurfranca** [ng5jr/nostr.build#3]
Hello, @ng5jr. As a file storage service author, could you please give your feedback here [https://github.com/nostr-protocol/nips/pull/547](https://github.com/nostr-protocol/nips/pull/547)?

**Comment by nostrbuild** [comment]
Hi @arthurfranca , we have migrated all of nostr.build to the below GitHub repository. @ng5jr has been implementing design updates, but the latest everything real-time is at https://github.com/nostrbuild/nostr.build

Regarding nip-95v2 feedback. We just implemented nip-98 authentication for file storage with nostr.build accounts. It is working great. https://github.com/fishcakeday is my partner and did the work.

We do not have plans for nip95 v2 etc. as it is a totally different file sharing architecture.
